BlobfishCollectionView/Router.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Router.swift
+//  Happy Blobfish
+//
+//  Created by Lukasz on 21/03/16.
+//  Copyright © 2016 Jan Grodowski. All rights reserved.
+//
+
+import Alamofire
+
+enum HappyBlobfishApi: URLRequestConvertible {
+    #if DEBUG
+    static let host: String = "http://localhost:9777"
+    #else
+    static let host: String = "http://www.mymagicserver.com:9777"
+    #endif
+    
+    case Index
+    case Upload
+    
+    var method: Alamofire.Method {
+        switch self {
+        case .Index:
+            return .GET
+        case .Upload:
+            return .POST
+        }
+    }
+    
+    var path: String {
+        switch self {
+        case .Index:
+            return "/memes"
+        case .Upload:
+            return "/uploads"
+        }
+    }
+    
+    var URLRequest: NSMutableURLRequest {
+        let URL = NSURL(string: HappyBlobfishApi.host)!
+        let mutableURLRequest = NSMutableURLRequest(URL: URL.URLByAppendingPathComponent(path))
+        mutableURLRequest.HTTPMethod = method.rawValue
+        return mutableURLRequest
+    }
+}
```

Comment:
Yeah, that is good article about enums. I also recommend to read official Swift docs about this concept https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html 

Enums in Swift are very powerful :)